### PR TITLE
include DB in projeсt

### PR DESCRIPTION
**Pull request by MaximBushuevExceed**

### CODE REVIEW

File: js.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  });
+
+  let result = await resp.json();
+  allTasks = result.data;
+  render();
+};
+
+const clickOnX = async (index) => {
+  const resp = await fetch(
+    `http://localhost:8000/deleteTask?_id=${allTasks[index]._id}`,
```
Comment (MaximBushuevExceed) [inline js.js:188]: destr

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,196 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans:ital,wght@0,400;0,700;1,400;1,700&family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;1,300;1,400;1,500;1,600&display=swap');
+
+*{
+  margin: 0px;
+  font-family: 'Open Sans', sans-serif;
+}
+
+.main{
```
Comment (MaximBushuevExceed) [inline styles.css:8]: code style

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  color: #fddca5;
+}
+
+.todo{
```
Comment (MaximBushuevExceed) [inline styles.css:16]: higher

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+#but:active{
+  background-color: #bbc8ba8f;
+}
+
+#tasks{
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  font-weight: 600;
+}
```
Comment (MaximBushuevExceed) [inline styles.css:72]: add empty line

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (MaximBushuevExceed) [inline styles.css:196]: remove empty

File: js.js [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+  allTasks = result.data;
+  render();
+};
+
+const updateValue = (event) => {
+  valueInput = event.target.value;
+};
+
+const onClickBut = async () => {
+  if (!(valueInput === "") && valueInput.length < 45) {
```
Comment (NataliaShchelkunova) [inline js.js:23]: why you check valueInput.length??

File: js.js [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+    });
+
+    let result = await resp.json();
+    allTasks = result.data;
+
+    valueInput = "";
+    input.value = "";
+    render();
+  } else {
+    alert("Введите значение менее 45 символов");
```
Comment (NataliaShchelkunova) [inline js.js:44]: why???? where is this task?

File: js.js [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+
+const render = () => {
+  const tasks = document.getElementById("tasks");
+
+  while (tasks.firstChild) {
+    tasks.removeChild(tasks.firstChild);
+  }
+
+  allTasks.sort((a, b) => {return a.isCheck-b.isCheck})
+
```
Comment (NataliaShchelkunova) [inline js.js:56]: remove empty

File: js.js [hunk 9]
```diff
@@ -0,0 +1,186 @@
...
+      img.appendChild(xButton);
+    }
+
+    tasks.appendChild(container);
+    container.appendChild(tasktext);
+    container.appendChild(img);
+    xButton.onclick = () => clickOnX(index);
+    pen.onclick = () => changeTask(item, index);
+  });
+
```
Comment (NataliaShchelkunova) [inline js.js:93]: remove

File: js.js [hunk 10]
```diff
@@ -0,0 +1,186 @@
...
+  changeBox.value = allTasks[index].text;
+  changeBox.type = "text";
+  taskedit.prepend(ok);
+  taskedit.prepend(dis);
+  taskedit.prepend(changeBox);
+  task.prepend(taskedit);
+  changeBox.addEventListener("change", changeValue);
+
+  ok.onclick = () => okClick(index, changeBox);
+  dis.onclick = () =>{
```
Comment (NataliaShchelkunova) [inline js.js:118]: whats happend? Why u use this function? For render? Why?

Comment (NataliaShchelkunova) [inline js.js:118]: codestyle

File: js.js [hunk 11]
```diff
@@ -0,0 +1,186 @@
...
+  } else {
+    alert("Нельзя оставить тоже самое или все удалить!");
+  }
+};
+
+changeValue = (event) => {
+  newText = event.target.value;
+};
+
+const checkFunction = async (check, index) => {
```
Comment (NataliaShchelkunova) [inline js.js:155]: where u use check? Why u give check?

File: js.js (hunk 1 above)
Comment (NataliaShchelkunova) [reply js.js:177]: !!!!!!!

File: js.js [hunk 12]
```diff
@@ -0,0 +1,180 @@
...
+};
+
+const render = () => {
+  const tasks = document.getElementById("tasks");
+
+  while (tasks.firstChild) {
+    tasks.removeChild(tasks.firstChild);
+  }
+
+  allTasks.sort((a, b) => { return a.isCheck - b.isCheck })
```
Comment (NataliaShchelkunova) [inline js.js:53]: what is it?

File: js.js [hunk 13]
```diff
@@ -0,0 +1,180 @@
...
+    allTasks = result.data;
+    newText = "";
+    changeBox.value = "";
+    render();
+  } else {
+    alert("Нельзя оставить тоже самое или все удалить!");
+  }
+};
+
+changeValue = (event) => {
```
Comment (NataliaShchelkunova) [inline js.js:145]: const

**Pull request merged**